test/unittest/bamboo_test.go

```diff
@@ -31,37 +30,8 @@ func TestBambooVariablesPopulatedWithValidValues(t *testing.T) {
 	assert.Equal(t, "https://atlassian.github.io/data-center-helm-charts", bambooAgent.AttributeValues["repository"])
 }
 
-func TestBambooDatasetImport(t *testing.T) {
-	t.Parallel()
-
-	datasetVars := BambooCorrectVariables
-	datasetVars["dataset_url"] = DatasetUrl
-	tfOptions := GenerateTFOptions(datasetVars, t, "products/bamboo")
-	plan := terraform.InitAndPlanAndShowWithStruct(t, tfOptions)
-
-	// verify Dataset import job
-	jobKey := "kubernetes_job.import_dataset[0]"
-	terraform.RequirePlannedValuesMapKeyExists(t, plan, jobKey)
-	job := plan.ResourcePlannedValuesMap[jobKey]
-	assert.Equal(t, "kubernetes_job", job.Type)
-
-	spec := job.AttributeValues["spec"].([]interface{})[0]
-	template := spec.(map[string]interface{})["template"].([]interface{})[0]
-	jobSpec := template.(map[string]interface{})["spec"].([]interface{})[0]
-	container := jobSpec.(map[string]interface{})["container"].([]interface{})[0]
-	commands := container.(map[string]interface{})["command"].([]interface{})
```

Comment:
We now using `random_string.random.result` to generate command and as far as the `random_string.random` resource is not defined in planning stage, the command is null and we can't test it now.